package.json

```diff
@@ -12,9 +12,9 @@
   ],
   "main": "index.js",
   "scripts": {
-    "lint": "eslint .",
+    "lint": "eslint . --ignore-path .gitignore",
```

Comment:
As I commented elsewhere, I don't understand the purpose of `--ignore-path .gitignore`. Why is it needed at all? If it is, then it should be added to `.eslintignore` file.